app/ide-desktop/lib/dashboard/src/components/UnstyledButton.tsx

```diff
@@ -26,10 +28,10 @@ export interface UnstyledButtonProps extends Readonly<React.PropsWithChildren> {
 
 /** An unstyled button with a focus ring and focus movement behavior. */
 function UnstyledButton(props: UnstyledButtonProps, ref: React.ForwardedRef<HTMLButtonElement>) {
-  const { focusRingPlacement, children, ...buttonProps } = props
+  const { noTooltip = false, focusRingPlacement, children, ...buttonProps } = props
```

Comment:
maybe we'll flip the logic? Pass `tooltip: string|JSX.Element|boolean` instead of `noTooltip`

1. It's much flexible
2. We can customize the tooltip text and separate it with aria-label (they are often orthogonal)
3. We can specify tooltip without specifying aria-label (button might have an implicit aria-label, or `aria-labelledby`)

Reply:
or i guess `React.ReactNode`

Reply:
ReactNode accepts null and undefined

Reply:
sure, but that's fine, isn't it?